src/main/java/org/gridsuite/study/server/dto/IndexingStatus.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.dto;
+
+/**
+ * @author Slimane amar <slimane.amar at rte-france.com
+ */
+public enum IndexingStatus {
```

Comment:
As indexing is done synchonously by import service, the status is now useless and should be removed?

Reply:
yes but in daily we said that we kept it anyway

Reply:
Removed